src/Type/EntityTypeManagerGetStorageDynamicReturnTypeExtension.php

```diff
@@ -23,10 +34,12 @@ class EntityTypeManagerGetStorageDynamicReturnTypeExtension implements DynamicMe
     /**
      * EntityTypeManagerGetStorageDynamicReturnTypeExtension constructor.
      *
+     * @param ReflectionProvider $reflectionProvider
      * @param string[] $entityTypeStorageMapping
      */
-    public function __construct(array $entityTypeStorageMapping = [])
+    public function __construct(ReflectionProvider $reflectionProvider, array $entityTypeStorageMapping = [])
```

Comment:
If you like to avoid the BC break I will have to add the reflection provider argument in second

Reply:
Nah, no need to avoid.